HW Submission from Luis Lithgow

LuisLithgow opened the pull request:
Incomplete..trouble with the footer

LuisLithgow committed:
Incomplete..trouble with the footer

qodesmith commented on css/style.css:
```diff
@@ -46,12 +46,52 @@ Developer @GA in NYC
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+     font-family: "open sans";
+     background: url(../images/header_bg.jpg) repeat top rgb(134,141,157);
```
:hammer: Don't forget our conversation about setting your html up first... good structure will lead to good css later. This image should only be on the "top" section of the page. You would need to re-think you're structure a bit to make it work.

LuisLithgow committed:
Fixed up the HTML structure

Re-did the HTML to a clear and understandable code. Afterwards, re-did
the CSS page to perfectly work with the HTML.

LuisLithgow commented:
@qodesmith Hey Aaron! This is a much better branch of HW#2. Let me know what you think about it.

LuisLithgow committed:
Minor change

Corrected some spelling on the index.HTML and changed a class name for
better practice.
